client/src/Clusters/Footer/Footer.style.css

```diff
@@ -0,0 +1,25 @@
+.footer-section {
+    border-top: 1px solid #7f7f7f;
+    margin: auto;
+    padding-top: 30px;
+    width: 90%;
+}
+
+.footer-left {
+    float: left;
```

Comment:
let's avoid float when possible. the styling here does not need to be done via float. floats tend to create unnecessary layout problems (maybe not now, but when you are making changes to layout in subsequent PRs).